requirements/requirements.in

```diff
@@ -23,8 +22,9 @@ setuptools-scm
 ipykernel
 ipywidgets
 spacy
+en_core_web_sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.4.1/en_core_web_sm-3.4.1.tar.gz
```

Comment:
I also reacted to your previous notes as I'm processing your review the same way you've processed the PR, immediate dispatch 😄 . 

I see what you're saying. I think this will bring less trouble than the previous solution. If it breaks, you should now immediately. At that moment you can always fetch the newest model without reinstalling sec-certs or anything like that.